include/bluetooth/mesh/light_ctrl_srv.rst

```diff
@@ -0,0 +1,427 @@
+Both the internal integral sum and the output level is represented as an unsigned 16-bit integer.
+
+To reduce noise, the regulator has a configurable accuracy property, which allows it to ignore errors smaller than the configured accuracy (represented as a percentage of the light level).
+See :option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_REG_ACCURACY` and :cpp:enumerator:`BT_MESH_LIGHT_CTRL_PROP_REG_ACCURACY <bt_mesh_light_ctrl::BT_MESH_LIGHT_CTRL_PROP_REG_ACCURACY>` for more information.
+
+.. note::
+   The illuminance regulator implementation only supports integers in its configuration.
+   The fractional part of coefficients, accuracy, and target levels is ignored.
+
+Sensor input
+------------
+
+The regulator relies on regular sensor input data to function correctly.
+This sensor data must come from an external :ref:`bt_mesh_sensor_srv_readme` model, reporting the ambient light level with the :cpp:var:`bt_mesh_sensor_present_amb_light_level` sensor type.
+The Sensor Server should publish its sensor readings to an address the Light LC Server is subscribed to, using a common application key.
+
+The Light LC Server will process all incoming sensor messages, and use them in the next regulator step. The regulator depends on frequent readings from the sensor server to provide a stable output for the Lightness Server. If the sensor reports are too slow, the regulator might oscillate, as it attempts to compensate for outdated feedback. It is strongly recommended to use the Sensor :ref:`bt_mesh_sensor_publishing_delta` feature for this purpose, triggering frequent reports when the regulator is compensating for large errors, while keeping the mesh traffic low in stable periods.
+
+The Sensor Server may be instantiated on the same mesh node as the Light LC Server, or on a different mesh node in the same area.
+The regulator performance depends heavily on the sensor's placement and sensitivity.
+In general, ambient light sensor devices should be placed such that their light sensor captures the human perception of the room's light level as closely as possible.
+
+Tuning the regulator
+--------------------
+
+You can tune the impact of each regulator component with coefficients:
+
+* K :sub:`p` - for the proportional component
+* K :sub:`i` - for the integral component
+
+These coefficients can have individual values for positive and negative errors, referenced to as follows in the API:
+
+* Kpu - proportional up; used when target is higher.
+* Kpd - proportional down; used when target is lower.
+* Kiu - integral up; used when target is higher.
+* Kid - integral down; used when target is lower
+
+Regulator tuning is a complex problem depending on a lot of internal external parameters.
+Varying sensor delay, light output and light change rate may significantly affect the regulator performance and accuracy.
+
+Regulators are typically tuned by analyzing their *step response*.
```

Comment:
How do the coefficients above relate to this step response?
The transition is a bit abrupt here: you write about coefficients, and then suddenly there is a change of subject.